macos/FluentUI/DatePicker/CalendarDayButton.swift

```diff
@@ -279,11 +279,7 @@ class CalendarDayButton: NSButton {
 						   constant: CalendarDayButton.dualModeMargin)
 
 	private var isDarkMode: Bool {
-		var isInDarkAppearance = false
-		if effectiveAppearance.bestMatch(from: [.darkAqua]) == .darkAqua {
-			isInDarkAppearance = true
-		}
-		return isInDarkAppearance
+		return NSAppearance.current.isDarkMode
```

Comment:
Keep using effectiveAppearance here.

Reply:
The appearance instance itself isn't used in determining light/dark theme. The underlying instance is only used in test mode. During application run time the theme information is read from `userdefaults` or `effectiveAppearance` based on iOS version. 

so swapping this with another instance won't have any effect.